anda/rust/rustup/rustup.spec

```diff
@@ -0,0 +1,62 @@
+Name:           rustup
+Version:        1.25.1
+Release:        %autorelease
+URL:            https://rust-lang.github.io/rustup/
+License:        MIT, Apache-2.0
+Summary:        The Rust toolchain installer
+Source0:        https://github.com/rust-lang/rustup/archive/refs/tags/%{version}.tar.gz
+BuildRequires:  cargo pkgconfig(openssl)
+
+%description
+Rustup installs The Rust Programming Language from the official release channels,
+enabling you to easily switch between stable, beta, and nightly compilers and keep them updated.
+It makes cross-compiling simpler with binary builds of the standard library for common platforms.
+
+
+%prep
+%setup -n rustup-%{version}
+
+
+%build
+cargo build --release --features no-self-update --bin rustup-init
+
+
+%install
+install -Dm755 target/release/rustup-init %{buildroot}/usr/bin/rustup
+
+_binlinks=('cargo' 'rustc' 'rustdoc' 'rust-gdb' 'rust-lldb' 'rls' 'rustfmt' 'cargo-fmt' 'cargo-clippy' 'clippy-driver' 'cargo-miri')
+cd %{buildroot}
+for link in "${_binlinks[@]}"; do
+    ln %{buildroot}/usr/bin/rustup usr/bin/${link}
+done
+
+# Generate completion files.
+mkdir -p %{buildroot}/usr/share/bash-completion/completions
```

Comment:
[standard macros](https://docs.fedoraproject.org/en-US/packaging-guidelines/RPMMacros/) should be used as well